packages/block-library/src/post-comments-form/view.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	getContext,
+	getElement,
+	store,
+	navigate,
+} from '@wordpress/interactivity';
+
+const focusableSelectors = [
+	'input:not([disabled]):not([type="hidden"]):not([aria-hidden])',
+	'select:not([disabled]):not([aria-hidden])',
+	'textarea:not([disabled]):not([aria-hidden])',
+	'[contenteditable]',
+	'[tabindex]:not([tabindex^="-"])',
+];
+
+const { state } = store( 'core/comments', {
+	state: {
+		error: '',
+		get showError() {
+			return state.error ? 'flex' : 'none';
+		},
+		get submitButtonText() {
+			const { isSubmitting } = getContext();
+			const { loadingText, submitText } = state;
+			return isSubmitting ? loadingText : submitText;
+		},
+		get displayCancelReply() {
+			const { formSlot } = getContext();
+			return formSlot ? undefined : 'none';
+		},
+	},
+	actions: {
+		submit: async ( event ) => {
```

Comment:
We still need to update this, see https://github.com/WordPress/gutenberg/pull/56984#discussion_r1427138184:

> This needs to use `*` instead of `async` and `yield` instead of `await` 🙂